src/env/env.rs

```diff
@@ -40,6 +44,19 @@ fn print_env(null: bool) {
     }
 }
 
+#[cfg(not(windows))]
+fn build_command( mut args: Vec<String> ) -> ( String, Vec<String> ) {
+    let command = args[0].clone();
+    args.remove(0);
+    ( command, args )
+}
+
+#[cfg(windows)]
+fn build_command( mut args: Vec<String> ) -> ( String, Vec<String> ) {
```

Comment:
Can you remove the spaces between the parentheses (applies in multiple spots)?

Reply:
Will do...

Reply:
Done.